test/WebJobs.Script.Tests/Configuration/HostJsonFileConfigurationSourceTests.cs

```diff
@@ -55,7 +56,7 @@ public void MissingHostJson_CreatesDefaultFile()
 
             AreExpectedMetricsGenerated(testMetricsLogger);
 
-            Assert.Equal(_defaultHostJson, File.ReadAllText(_hostJsonFile));
+            Assert.Equal(_hostJsonWithBundles, File.ReadAllText(_hostJsonFile));
 
             var log = _loggerProvider.GetAllLogMessages().Single(l => l.FormattedMessage == "No host configuration file found. Creating a default host.json file.");
             Assert.Equal(LogLevel.Information, log.Level);
```

Comment:
Could we add a test for _defaultHostJson too?

Reply:
Have updated the test to check that bundle configuration is case of readonly file system